clients/lsp-elixir.el

```diff
@@ -192,6 +192,7 @@ be available here: https://github.com/elixir-lsp/elixir-ls/releases/"
                                             "language_server.bat")
                                        ,@(cl-rest lsp-elixir-server-command))))
                   :activation-fn (lsp-activate-on "elixir")
+                  :major-modes '(elixir-mode elixir-ts-mode heex-ts-mode)
```

Comment:
nit: this might not be needed as `lsp-elixir` will be activated when the language is `elixir` (the `cdr` part of the cons you added bellow) already.

Reply:
thanks, makes sense. i will remove this now.